testing/btest/scripts/policy/protocols/conn/disable-unknown-ip-proto-support.zeek

```diff
@@ -0,0 +1,8 @@
+# @TEST-DOC: Test the policy for removing the unknown IP protocol field and connections from conn.log
+# @TEST-EXEC: zeek -C -r $TRACES/cisco-fabric-path.pcap %INPUT
```

Comment:
Just a detail but it looks like you don't actually need `-C` for that trace, here and below.

Reply:
Ah, I had pulled that invocation from the `core.cisco-fabric-path` test so I'll check/fix that one too.